### PR TITLE
winPB: Install NSClient++ On Windows Hosts For Nagios Monitoring

### LINKED ISSUE #2825: Enhancement: WindowsPB : Include Installation Of NSClient++ Software For Nagios Monitoring
- Request for new playbook addition

Install NSClient++ as part of the windows playbook/installation

Details:

In order to use nagios monitoring on windows machines, some software to provide the equivalent functionality to the nagios plugins used on linux/unix systems is required.  [NSClient++](https://www.nsclient.org/) is the most common piece of opensource windows software used to provide this functionality ( to report things such as disk space usage, cpu usage etc ). It may be possible to manually write scripts and plugins to achieve the same results, or identify individual plugins from the nagios plugin exchange, but having done some investigation, I believe this is a quicker solution to implement.

Implements Element 7.2 of Nagios Monitoring Epic #2445

### PR DESCRIPTION
**Pull request by steelhead31**
Fixes #2825 

Installs NSClient software for windows as part of the nagios monitoring improvements.

##### Checklist

- [X] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [X] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)

### CODE REVIEW

**Comment by steelhead31** [comment]
VPC run here: https://ci.adoptopenjdk.net/job/VagrantPlaybookCheck/1572/

**Comment by steelhead31** [comment]
New VPC run here ... https://ci.adoptopenjdk.net/job/VagrantPlaybookCheck/OS=Win2012,label=vagrant/1580/console

**Comment by steelhead31** [comment]
VPC check passed at https://ci.adoptopenjdk.net/job/VagrantPlaybookCheck/OS=Win2012,label=vagrant/1580/console

**Review by sxa — APPROVED** [review]

**Comment by karianna** [comment]
I see the CUDA download fails but I think that was fixed in another PR no?  Do we need to rebase?

**Comment by sxa** [comment]
> I see the CUDA download fails but I think that was fixed in another PR no? Do we need to rebase?

It's not fixed yet. Issue outstanding with NVidia.

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml [hunk 1]
```diff
@@ -11,3 +11,7 @@ Nagios_Plugins: Disabled
 ### Default JDK ###
 bootjdk: hotspot
 heapsize: normal
+
+## Nagios Server Details
+Nagios_Monitoring: Enabled
```
Comment (zdtsw) [inline ansible/playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml:16]: where this variable is used?

Comment (steelhead31) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml]: comment, it isnt, it was copied and pasted in error, so I've removed it

**Review by zdtsw — APPROVED** [review]

**Pull request merged**